packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -128,14 +154,29 @@ async function terraformAirnodeManage(
     ['var', 'handler_dir', handlerDir],
     ['input', 'false'],
     'no-color',
-    'auto-approve',
   ];
 
   if (httpGatewayApiKey) {
     commonArguments.push(['var', 'api_key', httpGatewayApiKey]);
   }
 
-  await execTerraform(execOptions, command, ...prepareAirnodeManageArguments(cloudProvider, commonArguments));
+  // Run import ONLY for an `apply` command (deployment). Do NOT run for `destroy` command (removal).
+  if (command === 'apply') {
+    const importOptions = cloudProviderAirnodeImportOptions[cloudProvider.name](cloudProvider as any);
+
+    if (!isEmpty(importOptions)) {
+      await execTerraform(
+        { ...execOptions, ignoreError: true },
+        'import',
+        prepareAirnodeManageArguments(cloudProvider, commonArguments),
+        cloudProviderAirnodeImportOptions[cloudProvider.name](cloudProvider as any)
+      );
+    }
+  }
+
+  commonArguments.push('auto-approve');
```

Comment:
What is this for?

Reply:
The `import` command uses the same arguments as the following `apply` command except for the `-auto-approve` one so I'm adding it after the `import` is called.